capstone-frontend/src/components/Login.jsx

```diff
@@ -1,80 +1,47 @@
 import React, { Component } from 'react'
-import { GoogleLogin, GoogleLogout } from 'react-google-login';
-import { Dropdown, DropdownButton, ButtonGroup } from 'react-bootstrap'
-import "../styles/Login.css"
-
+import { GoogleLogin } from 'react-google-login';
+import { Card, Navbar } from 'react-bootstrap'
+import '../styles/Login.css'
 
 export class Login extends Component {
 
-  constructor(props) {
-    super(props);
-
-    this.state = {
-      isLogin: false,
-      googleUser: {},
-      profileObj: {},
-      profileMenuCollapsed: false
-    };
-  }
-
-  toggleProfileMenu = () => {
-    this.setState({ profileMenuCollapsed: !this.state.profileMenuCollapsed });
-  }
-
   loginResponseSuccess = (response) => {
-    this.setState({ isLogin: true, googleUser: response, profileObj: response.profileObj });
 
     // Store User in Firebase
     fetch("/api/user", {
       method: "POST",
-      body: JSON.stringify(this.state.googleUser.tokenObj),
+      body: JSON.stringify(response.tokenObj),
     });
 
-    window.localStorage.setItem("userID", this.state.profileObj.googleId)
-
-  }
+    window.localStorage.setItem("userID", response.profileObj.googleId);
+    window.localStorage.setItem("profileObj", JSON.stringify(response.profileObj))
```

Comment:
What do we need the profileObj for later? And if we're already storing this, should we also store the userID separately (it's a field of profileObj)?

Reply:
The profileObj contains the user's info and profile image, which I used in my Logout component. I think storing the userID separately is good because we utilize it a lot in our fetches.